src/styles/main.scss

```diff
@@ -1,7 +1,338 @@
-@import "utils";
-@import "fonts";
-@import "typography";
+* {
+  box-sizing: border-box;
+}
+
+p,
+header,
+div,
+img,
+h1,
+h2,
+h3,
+h4,
+li,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+@mixin onTablet {
+  @media (min-width: 690px) {
+    @content;
+  }
+}
+
+@mixin onDesktop {
+  @media (min-width: 1170px) {
+    @content;
+  }
+}
+
+.page {
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+
+  &__body {
+    margin: 0 auto;
+  }
+}
+
+.header {
+  max-width: 1280px;
+
+  // max-width: 100%;
+  // height: 300px;
+  display: flex;
+  flex-direction: column-reverse;
+  // flex-direction: column-reverse;
+  margin: 0 auto px;
+
+  @include onTablet {
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    height: 384px;
+
+    @include onDesktop() {
+      display: grid;
+      grid-template-columns: repeat(12, 1fr);
+      height: 655px;
+    }
+  }
+
+  &__block {
+    padding: 0 20px;
+    width: 100%;
+    position: relative;
+
+    @include onTablet() {
+      padding: 45px 37px;
+    }
+    @include onDesktop() {
+      padding: 45px 58px;
+    }
+  }
+
+  &__menu {
+    display: block;
+
+    @include onTablet {
+      margin-bottom: 70px;
+    }
+    @include onDesktop() {
+      margin-bottom: 149px;
+    }
+
+    &--menu-opener {
+      height: 20px;
+      width: 20px;
+    }
+  }
+
+  &__title {
+    font-family: "Montserrat", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 45px;
+    line-height: 100%;
+    margin-bottom: 20px;
+
+    @include onTablet {
+      margin-bottom: 38px;
+    }
+    @include onDesktop() {
+      margin-bottom: 89px;
+    }
+  }
+
+  &__tickets {
+    display: flex;
+    flex-wrap: wrap;
+    width: 280px;
+    height: 50px;
+    border: 1px solid #1a5a4c;
+
+    @include onTablet() {
+      width: 270px;
+      height: 70px;
+    }
+    @include onDesktop() {
+      width: 270px;
+      height: 70px;
+      margin-left: 40%;
+    }
+
+    &-text {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      text-decoration: none;
+      color: #0f0e08;
+      font-family: "IBM Plex Sans", sans-serif;
+      font-weight: 500;
+      font-size: 16px;
+      line-height: 100%;
+      width: 85%;
```

Comment:
Why. do you use such a nesting here `&-`?

You could nest modifiers, or elements, but there you have non of them.

Watch more about naming convention in BEM [in this video](https://mate.academy/learn/css-advanced/bem#/video/920)
Watch closely when you use '-' in naming.